### ISSUE TITLE
adding lemmas

### ISSUE THREAD

**Issue by arademaker** [UniversalDependencies/UD_Portuguese-GSD#8]
I would like to add lemas in this corpus. Let us use this issue for discussing possible alternatives for that.

**Comment by dan-zeman** [comment]
Definitely! (Missing lemmas are another reason why I do not want to see the treebank in the shared task. We are evaluating lemmatization this year.)

I think you used a freely available morphological analyzer to re-tag (and lemmatize?) Bosque, right? Could you apply the same pipeline to GSD? Would disambiguation involve a lot of manual work?

**Comment by marcospln** [comment]
I could help with this task (not in the short-term, however).

**Comment by ceramisch** [comment]
I generated automatic LEMMAS + morphological FEATS obtained through UDPipe using 2.0 models trained on Bosque and applied to this GSD treebank, because it will be released as part of the MWE-annotated corpora in the PARSEME shared task.
But I'm not sure this would be a good starting point...

**Comment by marcospln** [comment]
Nice! I would say it's a good starting point. Maybe we could use an external dictionary (DELAF_PB?) to mark as ambiguous those entries (token+TAG) with more than one lemma. Thus we can focus on reviewing only the ambiguous ones.

**Comment by arademaker** [comment]
@ceramisch I know that UDPipe makes many mistakes in the morphology. It seems to be driven by the suffixes for all unknown word. But @marcospln idea could be useful. But you said that you generated the lemmas and feats, did you pushed your changes in the `dev` branch?

**Comment by ceramisch** [comment]
I like @marcospln 's idea too.

I didn't push the changes, I'm currently working on the PARSEME corpora release.

I will probably create another branch and push the changes by the end of the week, OK?

**Comment by arademaker** [comment]
Features must be revisied.

**Comment by arademaker** [comment]
We may use https://github.com/LFG-PTBR/MorphoBr
